### DISCUSSION TITLE
Move organization account to a personal accunt

### DISCUSSION THREAD

**Discussion by Thm5000** [community/community#123318] [Other Feature Feedback, Questions, & Ideas]
Hi, I have two accounts in Github, one personal and now Org account. All the repositories are in the org account. But i have a paied account on my personal account and the org account is free. How can I turn my personal account to an org account and use that and move all the repositories from my existing org account? Hope that was clear to understand. :) Hope someone can help me:)

**Comment by Rishit-katiyar** [comment]
okie sooo......First, go to your personal GitHub account and create a new organization. This will be like a new group for all your repositories. Give it a name that makes sense to you. Next, one by one, go to each repository you want to move and find the "Settings" tab. Look for the option to transfer the repository. Choose your new organization from the list and confirm .After transferring everything, you might need to adjust permissions or settings within your new organization to match your needs.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Thm5000 ! 

We’ve moved your post to our General category, which is more appropriate for this type of discussion.

Thanks for contributing @Thm5000 and @Rishit-katiyar !

**Reply by Rishit-katiyar** [reply]
hi @LiteBrite82 , are you a bot or a person who actually reads through hundreds of these question and corrects there category's and stuff. I am just curious

**Reply by LiteBrite82** [reply]
I am a living breathing human being; part of a small team that monitors and supports healthy community interactions 😄

**Reply by LiteBrite82** [reply]
We typically use scripted responses (written by us) for consistent messaging; when we can, we personalize them depending on the nature and depth of the question.

**Reply by LiteBrite82** [reply]
[This is us](https://github.com/orgs/community/discussions/28776)... oh snap! I haven't added myself to the list yet.

**Reply by Rishit-katiyar** [reply]
lets see , you sound like a bot and have a weather app in your repo list but you work for github . you are an interesting person

**Reply by LiteBrite82** [reply]
It's a pseudo account; we had doxing issues a while back. 

Sometimes I run and test code for folks in pages and programming help. The weather app was to troubleshoot a gh-pages deployment issue. 

I am definitely missing out on all the stats in my personal account though :(

**Reply by LiteBrite82** [reply]
fun fact: I chose litebrite82 because I was electrocuted by a litebrite when I was like.. 4? I bit the power cord; I still have a fat scar on my lower lip from it. My bot tone could be a side effect of the electrocution. And I am vintage 1982.

**Reply by Rishit-katiyar** [reply]
Again, you are very much an interesting person. electrocution is a part of life , I opened a cars and a refrigerator just to see its workings and got electrocuted both times .

**Reply by Rishit-katiyar** [reply]
By the way, what's the official GitHub repository name? I'd love to check out all the stuff you've been working on

**Comment by murtuja43** [comment]
Hey Thm5000, transferring organization content to a personal account on GitHub isn't directly supported. However, you can achieve a similar outcome by transferring ownership of the repositories to your personal account.

Here's a breakdown of the steps:

Create a Personal Account: If you don't have one already, sign up for a new personal account on GitHub.

Transfer Repository Ownership:

Go to the organization repository you want to move.
Navigate to the settings section (usually under a gear icon).
Look for options related to "Manage Access" or "Members."
Invite your new personal account to the organization with ownership permissions.
Once the invitation is accepted, ownership of the repository will be transferred.
Delete the Organization (Optional): After transferring all the desired repositories, you can choose to delete the organization if it's no longer needed.

**Comment by Thm5000** [comment]
Hi Md. Golam Murtuja Kayes,

Thank you for your answer. I Think I understand what you mean, but this is
the case. I can not change my personal account to an organization account
cause its already linked to this other organization account that we are
currently using.

So based on how these two accounts are linked and I have my paid
subscription on the personal account how can I then move everything from my
existing orgs account to my personal account and then change my
personal account to an orgs account?

You can see how it is today here: https://d.pr/i/wPxQB1

Thank you for helping me. :)
-- 

Med vennlig hilsen,


*Tor Helge Mattland*


CEO

Head of Digital Business Development / Co-founder

Product owner/Product Manager



*Tlf:* (+47) 934 89 693

*E-post: ***@***.*** ***@***.***>

*Nettside:* 4castmedia.no

*Adresse:* Karenslyst Allé 2 , 0278 Oslo




ᐧ

lør. 11. mai 2024 kl. 06:34 skrev Md. Golam Murtuja Kayes <
***@***.***>:

**Comment by LiteBrite82** [comment]
Hi @Thm5000 

This sounds complicated.. I want to make sure we understand each other.

- Personal accounts and Organizations are very different. A personal account represents you as an individual, where an organization is a collaborative space - like an office building where everyone goes to work. 

- There are also enterprise accounts which are used to control their associated organizations - much like a headquarters controls the individual stores of their brand (Paying for operational costs, providing standardization, opening and closing new branches). 

[Personal Accounts](https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts#personal-accounts) can be [Free](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-free-for-personal-accounts) or [Paid(GitHub Pro)](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro)depending on the tools you need as an individual developer. 
- A private account can also pay for add on tools like GitHub Copilot. 

[Organization Accounts](https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts#organization-accounts) can also be [Free](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-free-for-organizations) or [Paid(GitHub Team)](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-team) depending on the resource needs of the organization. 
- An organization can also pay for and provide extra tools like GitHub Copilot to its individual members (private accounts) - Much like how a company would provide work perks to its employees (company laptop, free coffee, office supplies). 
- Paid services provided to an individual (Personal Account) by an organization they belong to, last as long as the organization chooses to provide it. When an individual leaves the organization, they also lose any "perks" that were provide by the organization. 

As far as differences between what services come with a Personal Free account and an Organization Free account; there are not any except for the inherent team access controls for managing groups in an organization account. Your Personal Pro account has more perks than your Organization account. 

You would also want to have you own personal account be a member of your own organization. You would not want to work with the members of your team as the personification of the organization.

**Reply by LiteBrite82** [reply]
From my understanding of what you wrote. You want to have your Pro account perks in your organization?
